File: app/src/main/java/com/xunao/testlib/ToJSBean.java
package com.xunao.testlib;

public class ToJSBean {
    private String status;
    private String msg;
    private Object data;

    public ToJSBean(boolean success, Object data) {
        this.data = data;
        this.status = success ? "1" : "0";
        if(success) {
            this.msg = "成功";
        } else {
            if(data instanceof String) {
                this.msg = (String) data;
            } else {
                this.msg = "失败";
            }
        }

        this.msg = success ? "成功" : (String) data;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public Object getData() {
        return data;
    }

    public void setData(Object data) {
        this.data = data;
    }
}

File: library-udweb/src/main/java/com/uniondrug/udlib/web/bean/ToJSBean.kt
package com.uniondrug.udlib.web.bean

class ToJSBean(success: Boolean, var data: Any) {
    var status: String
    var msg: String

    init {
        status = if (success) "1" else "0"
        if (success) {
            msg = "成功"
        } else {
            if (data is String) {
                msg = data as String
            } else {
                msg = "失败"
            }
        }
        msg = if (success) "成功" else (data as String)
    }
}